media_driver/linux/common/codec/ddi/media_libva_encoder.cpp

```diff
@@ -378,8 +380,11 @@ VAStatus DdiEncode_CreateContext(
 
     MOS_STATUS eStatus = pCodecHal->Allocate(encCtx->m_encode->m_codechalSettings);
 
-#ifdef _MMC_SUPPORTED
+    //Get the priority from VAAPI.
```

Comment:
should be **_set_** the priority, right?

Reply:
Done.